src/Umbraco.Core/Strings/DefaultShortStringHelper.cs

```diff
@@ -566,7 +567,7 @@ internal void CopyTerm(string input, int ipos, char[] output, ref int opos, int
                     if (len > i)
                     {
                         term = term.Substring(i).ToLower(cultureInfo);
```

Comment:
Use Slice and https://learn.microsoft.com/en-us/dotnet/api/system.memoryextensions.tolower?view=net-6.0